src/loaders/compute_tools/checkm2/versions.yaml

```diff
@@ -1,7 +1,12 @@
+# reference_db_path: DIAMOND database CheckM2 relies on
+#                    download data following the instructions provided on https://github.com/chklovski/CheckM2#database
+
 versions:
   - version: 0.1.0
     date: 2023-07-19
+    reference_db_path: /global/cfs/cdirs/kbase/collections/libraries/CheckM2_database
```

Comment:
Rather than putting the full path in the versions.yml doc, why not put them under `ROOT_DIR` with a standard structure like everything else and just put the reference data version here?

Reply:
👍

Reply:
by the way, by doing that meaning each of us need to download the reference data in our personal repo if we want to run a mini test in our personal repo. I got `Disk quota exceeded` error when I tried to download the reference db to my personal repo. I can ask for more space or run test in the real root_dir. However, neither option is as convenient as simply specifying the full path in the 'versions.yml' document when doing test runs.

Reply:
Make a softlink in your personal space to the data in the kbase space

Reply:
👍 great. That works.

Reply:
Are there no versions for the checkm DB?

Reply:
there is none officially. It's just a diamond file `CheckM2_database/uniref100.KO.1.dmnd`. I can rename the directory name (CheckM2_database) to the current checkm2 version we are using?

Reply:
just found out that GTDB-TK needs to mount a whole directory (https://github.com/kbase/collections/blob/main/src/loaders/compute_tools/gtdb_tk/Dockerfile#L21)
while checkm2 needs to mount the diamond DB file (https://github.com/kbase/collections/blob/main/src/loaders/compute_tools/checkm2/Dockerfile#L26-L27)

that makes things more complicated.

Reply:
> I can rename the directory name (CheckM2_database) to the current checkm2 version we are using?

That seems reasonable - does the db change version to version?

Reply:
> > I can rename the directory name (CheckM2_database) to the current checkm2 version we are using?
> 
> That seems reasonable - does the db change version to version?

Since checkm2 is still pretty new. It doesn't yet. At least according to official github page
https://github.com/chklovski/CheckM2#database, it's always /path/to/database/CheckM2_database/uniref100.KO.1.dmnd

Reply:
Ok, well I think we can safely assume it will change. I guess the best we can do for now is the package version

Reply:
👍 updated default reference DB mount path.

Reply:
:+1: